src/particles/beam/BeamParticleContainerInit.cpp

```diff
@@ -790,6 +797,7 @@ InitBeamFromFile (const std::string input_file,
     std::string name_particle ="";
     std::string name_r ="", name_rx ="", name_ry ="", name_rz ="";
     std::string name_u ="", name_ux ="", name_uy ="", name_uz ="";
+    std::string name_s ="", name_sx ="", name_sy ="", name_sz ="";
```

Comment:
Can you add asserts after around line 980 that all of these have been found in the file and assigned to non-empty strings